scripts/cmake/targets.cmake

```diff
@@ -109,6 +109,25 @@ if(NOT WIN32)
     )
 endif()
 
+# ------------------------------------------------------------------------------
+# EFI
+# ------------------------------------------------------------------------------
+
+if(NOT WIN32 AND ENABLE_BUILD_EFI)
+
+    add_custom_target(efi_build
+        DEPENDS eapis_efi_${VMM_PREFIX}
```

Comment:
I dislike that this depends on build targets in the extended apis. Nothing in the base hypervisor should depend on anything in an extension.

Reply:
Agreed. Also, when I tried to build with no eapis enabled, I got a 'missing eapis_efi_x86_64-vmm-elf' make target error

Reply:
ENABLE_BUILD_EFI should default to off so hypervisor builds standalone normally, but enabling ENABLE_BUILD_EFI makes it so some eapis code is necessary.  Attempting to use the unmodified bfvmm in EFI would be pretty useless.  So, bfdriver EFI loader essentially relies on eapis.  Should the EFI loader itself be moved to extended apis?  extended_apis is only necessary if ENABLE_BUILD_EFI is on.

Reply:
It uses little eapis code and the extension could possibly live in bfdriver, just using base intrinsics instead of the eapis abstractions.  Just another possibility.